src/modules/fundingManager/bondingCurve/abstracts/RedeemingBondingCurveBase_v1.sol

```diff
@@ -280,6 +280,8 @@ abstract contract RedeemingBondingCurveBase_v1 is
     }
 
     /// @notice Virtual function to handle collateral tokens after a successful sell.
+    /// @dev    The downstream contract is responsible for implementing checks
+    ///         to ensure the contract has the right balance.
     /// @param  _receiver  The address for which the collateral tokens will be handled.
```

Comment:
Just saw this as well, I know you didn't do it, but as we touch this NatSpec section we may just fix that.
```suggestion
    /// @param  _receiver The address for which the collateral tokens will be handled.
```